### PR TITLE
Description Consolidation

### PR DESCRIPTION
**Pull request by Geevies**
I moved the desc_info to their parent thingies, because alt clicking the type and being sent to the description.dm was infuriating.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/clothing/chameleon.dm [hunk 1]
```diff
@@ -368,6 +372,12 @@
 			var/obj/item/gun/G = gun_type
 			src.gun_choices[initial(G.name)] = gun_type
 
+/obj/item/gun/energy/chameleon/change(picked in gun_choices) //Making the gun change its help text to match the weapon's help text.
```
Comment (alsoandanswer) [inline code/modules/clothing/chameleon.dm:375]: there is a /obj/item/gun/energy/chameleon/verb/change(picked in gun_choices) ahead, you should probably change the name of this

File: code/game/objects/items/weapons/melee/energy.dm [hunk 2]
```diff
@@ -208,9 +208,11 @@
  * Energy Sword
  */
 /obj/item/melee/energy/sword
-	color
```
Comment (alsoandanswer) [inline code/game/objects/items/weapons/melee/energy.dm:211]: don't remove this - it breaks code for some reason.

Comment (alsoandanswer) [reply code/game/objects/items/weapons/melee/energy.dm:211]: if you want to remove this anyway, remove the corresponding
/obj/item/melee/energy/sword/color
in the cargo datums/wherever it is.

Comment (alsoandanswer) [reply code/game/objects/items/weapons/melee/energy.dm:211]: if you want to remove it you have to remove /obj/item/melee/energy/sword/color in the code somewhere else so it compiles correctly.

File: code/modules/clothing/chameleon.dm [hunk 3]
```diff
@@ -366,7 +370,7 @@
 		gun_choices = list()
 		for(var/gun_type in typesof(/obj/item/gun/) - src.type)
 			var/obj/item/gun/G = gun_type
-			src.gun_choices[initial(G.name)] = gun_type
+			src.gun_choices[initial(G.name)] = gun_typ
```
Comment (alsoandanswer) [inline code/modules/clothing/chameleon.dm:373]:
```suggestion
			src.gun_choices[initial(G.name)] = gun_type
```

**Pull request merged**